assets/js/src/core/modules/asset/editor/types/document/tab-manager/tabs/preview/preview-container.tsx

```diff
@@ -11,20 +11,39 @@
 *  @license    https://github.com/pimcore/studio-ui-bundle/blob/1.x/LICENSE.md POCL and PCL
 */
 
-import React, { useContext } from 'react'
+import React, { useContext, useEffect, useState } from 'react'
 import { PreviewView } from './preview-view'
-import { useGetAssetByIdQuery } from '@Pimcore/modules/asset/asset-api-slice.gen'
 import { AssetContext } from '@Pimcore/modules/asset/asset-provider'
 import { ContentToolbarSidebarView } from '@Pimcore/modules/element/editor/tab-manager/layouts/content-toolbar-sidebar-view'
 
 const PreviewContainer = (): React.JSX.Element => {
   const assetContext = useContext(AssetContext)
-  const { data } = useGetAssetByIdQuery({ id: assetContext.id! })
+  const [docURL, setDocURL] = useState('')
+
+  useEffect(() => {
+    if (docURL !== '') {
+      return
+    }
+
+    fetch(`http://localhost/studio/api/assets/${assetContext.id!}/document/stream/pdf-preview`)
```

Comment:
Can we just use the relative path here? Without the domain?
Otherwise I would suggest that we create a new issue, to find out how we can make the domain configurable.

Reply:
see #453